src/js/TuningSliders.js

```diff
@@ -163,22 +179,29 @@ TuningSliders.initDTermFilterSliderPosition = function() {
 
 TuningSliders.resetPidSliders = function() {
     if (!this.cachedPidSliderValues) {
-        this.sliderMasterMultiplier = 1;
-        this.sliderRollPitchRatio = 1;
-        this.sliderIGain = 1;
-        this.sliderPDRatio = 1;
-        this.sliderPDGain = 1;
-        this.sliderDMinRatio = 1;
-        this.sliderFeedforwardGain = 1;
+        if (semver.lt(FC.CONFIG.apiVersion, API_VERSION_1_44)) {
+            this.sliderMasterMultiplier = 1;
+            this.sliderPDRatio = 1;
+            this.sliderPDGain = 1;
+            this.sliderFeedforwardGain = 1;
```

Comment:
This parametre can go outside this Api version condition because is aplied to all versions.

Reply:
Fixed